src/components/Tool/FixPhoneNumbers/FixPhoneNumbers.tsx

```diff
@@ -113,21 +99,34 @@ interface PersonPhoneNumbers {
   toDelete: PersonPhoneNumberInput[];
 }
 
+export interface ModalState {
+  open: boolean;
+  personId: string;
+  numberIndex: number;
+  phoneNumber: string;
+}
+
 interface Props {
   accountListId: string;
   setContactFocus: SetContactFocus;
 }
 
+const defaultDeleteModalState = {
```

Comment:
Right now the delete modal is looking pretty funky. It's would also be nice to know how the validation error message looks. Could you merge main into this PR?